hyperwallet/tests/test_client.py

```diff
@@ -252,6 +252,32 @@ def test_request_with_encryption_when_response_content_type_is_not_valid(self, s
             'Invalid Content-Type specified in Response Header'
         )
 
+    @mock.patch('requests.Session.request')
+    def test_request_with_encryption_when_status_code_204_and_content_type_is_not_set(self, session_mock):
+
+        data = {}
+
+        localDir = os.path.abspath(os.path.dirname(__file__))
+        clientPath = os.path.join(localDir, 'resources', 'private-jwkset1')
+        hyperwalletPath = os.path.join(localDir, 'resources', 'public-jwkset1')
+        encryption = Encryption(clientPath, hyperwalletPath)
+        encryptedMessage = encryption.encrypt(json.dumps(data))
+
+        session_mock.return_value = mock.MagicMock(
+            status_code=204,
```

Comment:
i dont know if we have style guide for python, but we should keep it consistent through the code(spacing before/after equals).